pkg/virt-operator/webhooks/kubevirt-create-admitter.go

```diff
@@ -58,6 +58,16 @@ func (k *kubeVirtCreateAdmitter) Admit(ctx context.Context, review *admissionv1.
 	}
 	//TODO: Do we want semantic validation
 
+	resp := webhookutils.NewPassingAdmissionResponse()
+	kv, _, err := getAdmissionReviewKubeVirt(review)
+	if err != nil {
+		return webhooks.ToAdmissionResponseError(err)
+	}
+
+	if len(kv.Spec.Configuration.DeveloperConfiguration.FeatureGates) > 0 {
+		resp.Warnings = append(resp.Warnings, fmt.Sprintf("kv.Spec.Configuration.DeveloperConfiguration.FeatureGates is deprecated. Please use kv.Spec.Configuration.DeveloperConfiguration.FeatureGatesMap instead"))
```

Comment:
1. redundant `fmt.Sprintf` - this is just a non parametrised string.
2. This long string is used twice - let's make it a constant.